src/test/ui/consts/const-points-to-static.rs

```diff
@@ -1,10 +1,10 @@
+// check-pass
 // compile-flags: -Zunleash-the-miri-inside-of-you
```

Comment:
This test should be moved to the miri-unleashed folder.

Reply:
Also it should then be adjusted to check that reading the static actually works. Like,
```rust
const TEST_VAL: u8 = *TEST;

fn main() { assert_eq!(TEST_VAL, 4); }
```